src/paths/all_shortest_paths.c

```diff
@@ -93,12 +95,12 @@
  * case.
  */
 
-igraph_error_t igraph_get_all_shortest_paths(const igraph_t *graph,
+igraph_error_t igraph_get_all_shortest_paths_cutoff(const igraph_t *graph,
                                   igraph_vector_int_list_t *vertices,
                                   igraph_vector_int_list_t *edges,
                                   igraph_vector_int_t *nrgeo,
                                   igraph_integer_t from, const igraph_vs_t to,
-                                  igraph_neimode_t mode) {
+                                  igraph_neimode_t mode, igraph_integer_t cutoff) {
```

Comment:
Can we make the cutoff be an `igraph_real_t` even in the unweighted version, for clarity? This is the choice I made in `igraph_distances()`, with the reasoning that passing `2.1` should include paths of length `2.0`. However, if the parameter is of integer type, then `2.1` is rounded to `2`, and will exclude paths of length `2`.